NewForce-Cohort-4/react-nutshell-house-baboonpaw
# Auth and Initialization

# Description
Added and tested the authorization components. 

Fixes # Added auth. 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Pull down the branch. If you have not previously cloned the repo and created your own local api/database.json, you will need to do so.
- You might have to run npm install. I'm not sure if those files are saved here.
- You will need to start your React app with "npm start"
- You will need to start your json-server on "-p 8088"
- Once the app loads in the browser, you should see a login form.
- Try to login with your email. You should see a pop-up that informs you that your email is not yet registered.
- Click the "Not a member yet?" to register. Register as a user.
- After successful registration, the webpage should load to a blank page with a NavBar with blue text and black background. 

# Checklist:
